static/elements/form-field-specs.js

```diff
@@ -81,19 +89,26 @@ export const ALL_FIELDS = {
 
   'impl_status_chrome': {
     help_text: html`
-        Implementation status in Chromium.`,
+        Select the appropriate Chromium development stage. If you
+        select In developer trial, Origin trial, or Enabled by
+        default, be sure to set the equivalent Process stage.`,
   },
 
   'bug_url': {
     help_text: html`
         Tracking bug url (https://bugs.chromium.org/...). This bug
         should have "Type=Feature" set and be world readable.
-        Note: This field only accepts one URL.`,
+        Note: This field only accepts one URL.
+        <br/><br>
```

Comment:
Any reason why one of these BR's has the slash and the other does not?  It is a self-closing tag regardless.

Reply:
I thought I used `<br/>` for both.  There used to be a good reason to use `<br/>` but maybe it no longer matters.
But it is generally better to avoid br altogether and instead use block tags around neighboring text, so the extra whitespace is inferred.  That's more work :)